### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by solveigag**
Amended error handling to accommodate two additional sad path tests

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -1,87 +1,199 @@
...
+          });
+        });
+    });
+  });
+  describe("STATUS 400s", () => {
+    test("Status 400 - invalid id", () => {
+      return request(app)
+        .get("/api/articles/invalidId")
+        .expect(400)
+        .then((res) => {
```
Comment (alexdevdavis) [inline __tests__/app.test.js:78]: if you're destructuring body from response in earlier tests, try to be consistent with your decision to do so

File: __tests__/app.test.js [hunk 2]
```diff
@@ -1,87 +1,199 @@
...
+        });
+    });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  describe("STATUS 200", () => {
+    test("Responds with 200 and returns relevant article with incremented votes", () => {
+      const newVote = { inc_votes: 1 };
+      const date = new Date(1594329060000).toJSON();
```
Comment (alexdevdavis) [inline __tests__/app.test.js:97]: the purpose of creating a date as part of this test is unclear to me

File: __tests__/app.test.js [hunk 3]
```diff
@@ -1,87 +1,199 @@
...
+        .send(newVote)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.updatedArticle).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: date,
```
Comment (alexdevdavis) [inline __tests__/app.test.js:128]: think carefully about which properties are actually relevant to your tests; to what extent is your code DRY here?

File: app.js [hunk 4]
```diff
@@ -22,7 +26,7 @@ app.use((err, req, res, next) => {
 })
 
 app.all("/*", (req, res) => {
```
Comment (alexdevdavis) [inline app.js:28]: I'd suggest keeping your endpoint functions together and having your error handling functions come afterwards

**Review by alexdevdavis — APPROVED** [review]

File: __tests__/app.test.js (hunk 2 above)
Comment (solveigag) [reply __tests__/app.test.js:97]: @alexdevdavis  If I copy the long number in the object I provide test fails as it takes those numbers literally

**Pull request merged**